workshop/exercise-7/README.md

```diff
@@ -90,9 +90,9 @@ Note that `cert-chain.pem` is Envoy’s public certificate (i.e., presented to t
 
 If, for some reason, mTLS is not functional, the Guestbook application will not operate correctly. This would be evident in its UI (e.g., it would display a [`Waiting for database connection`](waiting_on_database.png) message). In that case, mTLS should be disabled.
 
-mTLS it can be disabled globally or per service.
+mTLS can be disabled globally or per service.
 
-* To disable mTLS for all services in the mesh, comment out the `authPolicy` settings in the mesh configuration:
+* To disable mTLS for all services in the mesh, comment out the `authPolicy` settings in the mesh configuration, and then restart Istio Pilot to pick up the new configuration:
```

Comment:
This instruction doesn't work.  It used to work in older istio release (like 0.2 or 0.3) but Dan Berg tried it today and didn't work on 0.7.1

Reply:
Works with 0.6 which was the target before KubeCon, IIRC. Will fix for 0.7.1

Reply:
I will do this as part of PR to #39